src/sms-verification/templates/user_management/sms-verification/verify_form.html

```diff
@@ -16,9 +16,14 @@
     <div class="help-block" ng-if="errors.activation_code" ng-bind="errors.activation_code"></div>
 </div>
 
+<!-- This module does not parse an array of errors into a single error string
+    so we must loop over the property on scope.errors. -->
 <div ng-show="errors.non_field_errors" ng-class="{'has-error': errors.non_field_errors}">
     <div ng-repeat="error in errors.non_field_errors" class="help-block" ng-bind="error"></div>
```

Comment:
Array: https://github.com/incuna/angular-user-management/blob/ca68ae97af13cb30fb3b58644075bbf38aba085f/src/sms-verification/scripts/directives.js#L34

vs

Single: https://github.com/incuna/angular-user-management/blob/ca68ae97af13cb30fb3b58644075bbf38aba085f/src/sms-password/scripts/directives.js#L35-L43

The former is the only instance in this library: it should be updated to be like the latter.